cmd/edgex-ui-server/static/js/pages/devices/deviceService.js

```diff
@@ -358,7 +535,7 @@ orgEdgexFoundry.deviceService = (function(){
 		//debugger
 		var device = {
 			service: {
-				name: $(".edgexfoundry-device-form input[name='deviceServiceName']").val().trim(),
+				name: $("#deviceServiceNameSelect").val(),
```

Comment:
please set the variable as prefixed by the property value in edgexfoundry in next time